eox_nelp/pearson_vue/tests/test_pipeline.py

```diff
@@ -597,29 +597,31 @@ def tearDown(self):
         """Restore mocks' state"""
         CourseEnrollment.reset_mock()
 
+    @patch.object(timezone, "now")
     @override_settings()
-    def test_get_exam_data_success(self):
+    def test_get_exam_data_success(self, mock_now):
```

Comment:
This is not testing your changes... indeed if you revert the pipeline changes this test passes